diagnostic_aggregator/include/diagnostic_aggregator/aggregator.hpp

```diff
@@ -120,41 +122,51 @@ class Aggregator
   /*!
    *\brief True if the NodeHandle reports OK
    */
-  bool ok() const { return n_.ok(); }
+  bool ok() const {return rclcpp::ok();}
 
   /*!
    *\brief Publish rate defaults to 1Hz, but can be set with ~pub_rate param
    */
-  double getPubRate() const { return pub_rate_; }
+  double getPubRate() const {return pub_rate_;}
+
+  rclcpp::Node::SharedPtr get_node();
```

Comment:
can this function be const?

Reply:
not sure, will try that

Reply:
Since aggregator will become a node now (instead of just owning one), I think about only exposing its parameter interface via API. What do you think?